duckdb_engine/__init__.py

```diff
@@ -207,7 +207,7 @@ def get_view_names(
         include: Any = ...,
         **kw: Any,
     ) -> Any:
-        s = "SELECT name FROM sqlite_master WHERE type='view' ORDER BY name"
-        rs = connection.exec_driver_sql(s)
+        s = f"SELECT table_name FROM information_schema.tables WHERE table_type='VIEW' and table_schema=?"
```

Comment:
Is filter condition:  table_type='VIEW'  required?  since it is also called from superset - list table function, in such case,  no need this filter or  specify  table_type= 'Base Table'